example/lib/features/country/domain/country_entity.dart

```diff
@@ -8,29 +8,39 @@ class CountryEntity extends Entity {
     this.countries = const [],
     this.continents = const {},
     this.selectedContinentId = 'AS', // AS is country code for Asia.
+    this.errorMessage = '',
```

Comment:
Is saving an  error message as string an approach we are going to recommend doing? or it is done like this just for the sake of example?